tests/behat/features/capabilities/group/group-create-secret.feature

```diff
@@ -0,0 +1,182 @@
+    Then I should not see "Secret group"
+    Given I am logged in as "SecretGroup User One"
+    And I am on "user"
+    And I click "Groups"
+    And I click "Add group"
+    Then I click radio button "Secret group This is a secret group. Users can only join by invitation and the group itself and its content are hidden from non members." with the id "edit-group-type-secret-group"
+    And I press "Continue"
+    When I fill in "Title" with "Test secret group"
+    And I fill in the "edit-field-group-description-0-value" WYSIWYG editor with "Description text"
+    And I fill in "Location name" with "Disclosed"
+    And I select "NL" from "Country"
+    And I wait for AJAX to finish
+    Then I should see "City"
+    And I fill in the following:
+      | City | Hengelo |
+      | Street address | Padangstraat 11 |
+      | Postal code | 7556SP |
+    And I press "Save"
+    And I should see "Test secret group" in the "Main content"
+    And I should see "Disclosed"
+    And I should see "1 member"
+    And I should see "Joined"
+    And I should not see the link "Read more"
+
+    And I should see "Test secret group" in the "Hero block"
+    And I should see the button "Joined"
+    And I should see the link "Edit group" in the "Hero block"
+    And I should see "Disclosed" in the "Hero block"
+    And I should see "Padangstraat 11" in the "Hero block"
+    And I should see "7556SP" in the "Hero block"
+
+  # Create a post inside the secret group, visible to group members only
+    When I click "Stream"
+    And I fill in "Say something to the group" with "This is a secret group post."
+    And I select post visibility "Group members"
+    And I press "Post"
+    Then I should see the success message "Your post has been posted."
+    And I should see "This is a secret group post."
+
+  # Create a topic inside the secret group
+    When I click "Topics"
+    And I should see the link "Create Topic" in the "Sidebar second"
+    And I click "Create Topic"
+    When I fill in the following:
+      | Title | Test secret group topic |
+    And I fill in the "edit-body-0-value" WYSIWYG editor with "Body description text"
+    And I click radio button "Discussion"
+    And I press "Save"
+    And I should see "Test secret group topic"
+
+  # Create an event inside the secret group
+    And I click "Test secret group"
+    When I click "Events"
+    And I should see the link "Create Event" in the "Sidebar second"
+    And I click "Create Event"
+    And I fill in the following:
+      | Title | Test secret group event |
+      | edit-field-event-date-0-value-date | 2025-01-01 |
+      | edit-field-event-date-end-0-value-date | 2025-01-01 |
+      | Time  | 11:00:00    |
+      | Location name       | Technopark |
+    And I fill in the "edit-body-0-value" WYSIWYG editor with "Body description text."
+    And I press "Save"
+    And I should see "Test secret group event"
+
+  # Lets add another user on the Manage members tab.
```

Comment:
I think we can make this a separate scenario, starting with "Given secret group...."

We also don't actually test the visibility here but only that creation works.